src/cli/query.rs

```diff
@@ -95,6 +102,24 @@ fn query(
 
     let statement: String = call.req(engine_state, stack, 0)?;
 
+    let params: Option<serde_json::Value> =
+        match call.get_flag::<Value>(engine_state, stack, "params")? {
+            Some(p) => match p {
+                Value::Record { .. } => Some(convert_nu_value_to_json_value(&p, span).unwrap()),
+                Value::List { .. } => Some(convert_nu_value_to_json_value(&p, span).unwrap()),
+                _ => {
+                    return Err(ShellError::GenericError {
+                        error: "Parameters must be a list or record".to_string(),
```

Comment:
Maybe we should use object rather than record, users are unlikely to know the nu lingo.

Reply:
Good point, will change this now